rebar.config

```diff
@@ -37,3 +37,5 @@
            ]}.
 
 {post_hooks, [{clean, "rm -f .dialyzer_plt"}]}.
+
+{pre_hooks, [{clean, "rm -rf erl_crash.dump *.log *.coverage.xml"}]}.
```

Comment:
I don't think this repo outputs `*.coverage.xml`, doesn't look like it's set up for coverage reporting at all TBH.

Reply:
Removed coverage pattern